user/storage.md

```diff
@@ -0,0 +1,267 @@
+  are compatible with S3.
+- **[gcsfs](https://gcsfs.readthedocs.io/en/latest/)** - for working with Google
+  Cloud Storage.
+- **[adlfs](https://github.com/fsspec/adlfs)** - for working with Azure Data Lake
+  and Azure BLOB Storage.
+
+Each system has its own unique mechanisms for authentication and authorization;
+consult the documentation links above for more details.
+
+#### Reading Data
+
+When reading data from cloud object storage, you have two general options:
+- Download the data to the local filesystem; this is fine for small data, but not suitable
+  large data or cloud-optimized datasets. Downloads can be managed with
+  [Pooch](https://www.fatiando.org/pooch/latest/) or fsspec.
+- Open the data with an application that understands how to stream data data
+  over HTTP directly from object storage. This is suitable for large data and
+  cloud-native formats.
+
+As an example of the latter use case, here is how you would open the
+[NASA  Multi-Scale Ultra High Resolution (MUR) Sea Surface Temperature (SST)](https://registry.opendata.aws/mur/)
+dataset from the AWS Public Data program using Xarray:
+
+```python
+import xarray as xr
+ds = xr.open_dataset("s3://mur-sst/zarr/", engine="zarr", storage_options={"anon": True})
+```
+
+#### Writing Data
+
+Writing data (and reading private data) requires credentials for authentication.
+2i2c does not provide credentials to individual users.
+Instead you 2i2c customers should manage their own cloud storage directly.
+
+On S3-type storage, you will have a client key and client secret associated with you account.
+The following code creates a writeable filesystem:
+
+```python
+import s3fs
+fs = s3fs.S3FileSystem(key='<YOUR_CLIENT_KEY>', secret='<YOUR_CLIENT_SECRET')
+```
+
+Non-AWS S3 services (e.g. Wasabi Cloud) can be configured by passing an argument
+such as `client_kwargs={'endpoint_url': 'https://s3.us-east-2.wasabisys.com'}`
+to `S3FileSystem`.
+
+For Google Cloud Storage, the best practice is to create a
+[service account](https://cloud.google.com/iam/docs/service-accounts) with
+appropriate permissions to read / write your private bucket.
+You upload your service account key (a .json file) to your hub
+home directory and then use it as follows:
+
+```python
+ import json
+ import gcsfs
+ with open('<your_token_file>.json') as token_file:
+     token = json.load(token_file)
+ gcs = gcsfs.GCSFileSystem(token=token)
+```
+
+You can then read / write private files with the ``gcs`` object.
+
+### Scratch Bucket
+
+Some 2i2c environments are configured with a "scratch bucket," which
+allows you to temporarily store data. Credentials to write to the scratch
+bucket are pre-loaded into your Pangeo Cloud environment.
+
+:::{warning}
+Any data in scratch buckets will be deleted once it is 7 days old.
+Do not use scratch buckets to store data permanently.
+:::
+
+The location of your scratch bucket is contained in the environment variable ``PANGEO_SCRATCH``.
```

Comment:
This is probably something other than `PANGEO_SCRATCH` - what is it?

Reply:
On our 2i2c-managed clusters, I believe it is still PANGEO_SCRATCH. But I'd be happy to see it renamed.

Reply:
SCRATCH_BUCKET and PANGEO_SCRATCH are both set. I think we should recommend SCRATCH_BUCKET